File: Steven/src/main/java/parser/nodes/expression/GreaterThan.java
package parser.nodes.expression;

import parser.Visitor;

/**
 * Created by Steven Kok on 21/02/2015.
 */
public class GreaterThan extends BinaryExpression {
    public GreaterThan(Expression left, Expression right) {
        super(left, right);
    }
    @Override
    public void accept(Visitor visitor) {
        visitor.visit(this);
    }
}

File: FelipezConde/QuestionnaireLanguage/AST/Nodes/Expressions/Binaries/GreaterThan.cs

namespace AST.Nodes.Expressions.Binaries
{
    public class GreaterThan : Binary
    {
        
        public GreaterThan(Expression left, Expression right, PositionInText position)
            : base(left, right, position)
        {}

        public override T Accept<T>(VisitorInterfaces.IExpressionVisitor<T> visitor)
        {
            return visitor.Visit(this);
        }

        public override string ToString()
        {
            return ">";
        }
    }
}
